examples/panorama.html

```diff
@@ -31,35 +36,37 @@
         <script src="../dist/debug.js"></script>
         <script src="GUI/dat.gui/dat.gui.min.js"></script>
         <script type="text/javascript">
-            /* global itowns, document, renderer */
             var viewerDiv;
             var view;
 
             viewerDiv = document.getElementById('viewerDiv');
 
-            const urls = [
+            // Declare 5 panoramas
+            var urls = [
```

Comment:
Even if the way to prepare the data/metadata looks quite natural, it would be great to have a readme on it (here for example https://github.com/iTowns/iTowns2-sample-data/tree/master/panoramas)